man/mkdocs/overrides/partials/footer.html

```diff
@@ -87,7 +87,7 @@
   <!-- Further information -->
    <div class="md-footer-meta md-typeset">
        <center>
-      <a href="./index.html">Main index</a> | <a href="./topics.html">Topics index</a> | <a href="./keywords.html">Keywords index</a> | <a href="./graphical_index.html">Graphical index</a> | <a href="./full_index.html">Full index</a><br>
+      <a href="./index.html">Main index</a> | <a href="./topics.html">Topics index</a> | <a href="./tags.html">Tags index</a> | <a href="./graphical_index.html">Graphical index</a> | <a href="./full_index.html">Full index</a><br>
```

Comment:
Let's put it alongside keywords.html now instead of deleting it. Picking between the two can be part of a broader evaluation of the new doc.